app/services/idv/steps/recover_doc_success_step.rb

```diff
@@ -0,0 +1,28 @@
+module Idv
+  module Steps
+    class RecoverDocSuccessStep < DocAuthBaseStep
+      def call
+        pii_from_doc = session['idv/recovery']['pii_from_doc']
+        decrypted_pii = JSON.parse(saved_pii)
+        return unless pii_matches_data_on_file?(pii_from_doc, decrypted_pii)
+
+        mark_step_complete(:recover_fail)
+        redirect_to account_url
+        session['need_two_factor_authentication'] = false
+      end
+
+      private
+
+      def saved_pii
+        session['decrypted_pii']
```

Comment:
Not tested yet. Makes me wonder if there's something overriding this since line 6 above looks like it calls this.

Reply:
added coverage